shared/lib-metrics/src/main/java/gov/va/vro/metricslogging/MetricLoggerService.java

```diff
@@ -19,6 +20,21 @@ public class MetricLoggerService implements IMetricLoggerService {
 
   private final MetricsApi metricsApi;
 
+  @Value("${vro.env}")
+  public final String env;
+
+  @Value("${vro.it-portfolio:benefits-delivery}")
+  public final String itPortfolio;
+
+  @Value("${vro.team:va-abd-rrd}")
+  public final String team;
+
+  @Value("${vro.app.name}")
+  public final String service;
+
+  @Value("${vro.app.dependency:")
+  public final String dependency;
```

Comment:
This could potentially be a list of tags like dependency:bip_claims_api, dependency:bip_evidence_api. So really this should maybe be a list of dependencies.

Reply:
Interesting, I didn't interpret the documentation as allowing a collection.  I'll turn this into a collection.